README.md

```diff
@@ -123,6 +123,14 @@ It is recommended to have this rule enabled in most cases.
 ## Role assigned
 This rule only accept offers assigned to the Role defined in `mesos.role`.
 
+## Framework Authentication/Authorisation
+To use [Framework Authentication](http://mesos.apache.org/documentation/latest/authentication/), please pass the following settings:
+
+| Command | Description |
```

Comment:
Maybe a column on wether it's optional or not?

And if we want to take this even further as a template maybe a column on default value.